po/ko.po

```diff
@@ -434,15 +437,15 @@ msgstr ""
 
 #: src/login/org.freedesktop.login1.policy:289
 msgid "Suspend the system while an application is inhibiting this"
-msgstr "프로그램이 절전 상태 진입을 못하게 요청할 때 시스템 절전 상태 진입"
+msgstr "프로그램에서 대기 모드 진입을 막을 때 시스템 대기 상태 진입"
 
 #: src/login/org.freedesktop.login1.policy:290
 msgid ""
 "Authentication is required to suspend the system while an application is "
 "inhibiting this."
 msgstr ""
-"프로그램이 절전 상태 진입을 못하게 요청할 때 시스템을 절전 상태로 놓으려면 인"
-"증이 필요합니다."
+"프로그램에서 대기 모드 진입을 막을 때 시스템을 대기 상태로 설정하려면 인증이 "
+"필요합니다."
```

Comment:
Hmm, translate.google.com tells me this changes "sleep state" to "standby". Is the new form something that is used commonly, consistent with the strings used e.g. in gnome-shell?

Reply:
Yes, that's right. refer: https://gitlab.gnome.org/GNOME/gnome-tweaks/-/blob/5a43242bca3963433cefecc492fab9b772454d47/po/ko.po#L309